scripts/go2.lic

```diff
@@ -161,6 +164,30 @@ show_help = proc {
    respond output
 }
 
+# For Dragonrealms only
+# A change in behavior from Lich4 to Lich5 caused certain ;go2 map wayto movements to fail.
+# This is because of the way the map stringprocs are structured, and the method used to evaluate
+# certain map moves, which involved an arguably dubious overloading of FalseClass on Lich4.
+# The transition to Lich5 makes it necessary to fix some stringprocs as an interim solution, at
+# least until the map can be divested from current owner control and handled appropriately. This code
+# also allows the flexibility to pull custom map wayto overrides from the standard yaml hierarchy.
+if XMLData.game =~ /^DR/
+  # Get yaml settings
+  settings = get_settings
```

Comment:
In case someone doesn't have dependency DR running, should this all be wrapped in `if defined?(get_settings)` as to avoid breaking someone that has yet to install dependency?